src/main/java/lotto/view/ResultView.java

```diff
@@ -15,39 +18,55 @@ public class ResultView {
     private static final String LOTTO_PRINCIPAL_RATE_INFORMATION = "(기준이 1이기 때문에 결과적으로 본전이라는 의미임)";
     private static final String LOTTO_PROFIT_RATE_INFORMATION = "(기준이 1이기 때문에 결과적으로 이득이라는 의미임)";
     private static final String BLANK = "";
+    private static final String NEXT_LINE = "\n";
     private static final String HORIZONTAL_LINE = "----------";
+    private static final int RATE_CRITERIA = 1;
 
-    public int lottoCount(int purchaseAmount) {
-        return purchaseAmount / LOTTO_PRICE;
-    }
-    public void showLottoCount(int lottoCount) {
-        System.out.println(lottoCount + INFORMATION_PURCHASE_LOTTO_COUNT);
+    private ResultView(){}
+
+    public static void showLottoCount(int purchaseCount) {
+        System.out.println(purchaseCount + INFORMATION_PURCHASE_LOTTO_COUNT);
         System.out.println(BLANK);
     }
 
-    public void showLottoList(Lottos lottos) {
-        for (int i = 0; i< lottos.getSize(); i++) {
-            System.out.println(lottos.getLotto(i).getLotto());
+    public static void showLottoList(Lottos lottoList) {
+        for (int i = 0; i< lottoList.getSize(); i++) {
+            System.out.println(lottoList.getLotto(i).getNumbers());
         }
     }
 
-    public void showLottoStatistics(LottoStatistics lottoStatistics) {
-        System.out.println(BLANK);
-        System.out.println(INFORMATION_LOTTO_STATISTICS);
-        System.out.println(HORIZONTAL_LINE);
-        for (LottoMatchInformation matchInformation : lottoStatistics.getInformation().values()) {
-            System.out.println(String.format(LOTTO_MATCH_INFORMATION, matchInformation.getMatch(), (int) matchInformation.getPrize(), matchInformation.getCount()));
+    private static StringBuilder getInformationStatistics() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(NEXT_LINE);
+        sb.append(INFORMATION_LOTTO_STATISTICS);
+        sb.append(NEXT_LINE);
+        sb.append(HORIZONTAL_LINE);
+        return sb;
+    }
+
+    public static void showLottoStatistics(LottoMatchNumbers lottoMatchNumbers, LottoStatistics lottoStatistics) {
+        System.out.println(getInformationStatistics());
+        Map<Integer, Integer> matchNumberMap = lottoMatchNumbers.getMatchNumberMap();
+        for (Integer key : matchNumberMap.keySet()) {
+            System.out.println(
+                    String.format(
+                            LOTTO_MATCH_INFORMATION,
+                            lottoStatistics.getInformation().get(key).getMatch(),
+                            (int) lottoStatistics.getInformation().get(key).getPrize(),
+                            matchNumberMap.get(key)));
         }
-        double rate = lottoStatistics.getRate();
+    }
+
+    public static void showRate(double rate) {
```

Comment:
직접 판단하는걸 더 최소화 시켜도 좋을 것 같아요. 
```suggestion
    public static void showRate() {
        RateType rateType = RateType.valueOf(rate);
        System.out.println(rateType.getMessage());
```